extensions/2.0/Vendor/EXT_mesh_features/README.md

```diff
@@ -127,263 +141,509 @@ For example
+
+> **Implementation note:** Use of floating-point properties in a property texture would require a floating point-compatible image format like KTX2, provided by an additional extension.
+
+> **Example:** Property texture implementing a "wall" class, with property values stored in a glTF texture at index 0 and indexed by `TEXCOORD_0`.
+>
+> <img src="figures/feature-texture.png"  alt="Property Texture" width="500">
+>
+> _Class and property texture_
+>
+> ```jsonc
+> {
+>   "extensions": {
+>     "EXT_mesh_features": {
+>       "schema": {
+>         "classes": {
+>           "wall": {
+>             "name": "Wall Temperature vs. Insulation",
+>             "properties": {
+>               "insideTemp": {
+>                 "name": "Inside Temperature",
+>                 "type": "UINT8"
+>               },
+>               "outsideTemp": {
+>                 "name": "Outside Temperature",
+>                 "type": "UINT8"
+>               },
+>               "insulation": {
+>                 "name": "Insulation Thickness",
+>                 "type": "UINT8",
+>                 "normalized": true
+>               },
+>             }
+>           }
+>         }
+>       },
+>       "propertyTextures": [{
+>         "class": "wall",
+>         "index": 0,
+>         "texCoord": 0,
+>         "properties": {
+>           "insideTemp": [0],
+>           "outsideTemp": [1],
+>           "insulation": [2]
+>         }
+>       }]
+>     }
+>   }
+> }
+> ```
+>
+> _Primitive_
+>
+> ```jsonc
+> {
+>   "primitives": [
+>     {
+>       "attributes": {
+>         "POSITION": 0,
+>         "TEXCOORD_0": 1
+>       },
+>       "indices": 2,
+>       "material": 0,
+>       "extensions": {
+>         "EXT_mesh_features": {
+>           "propertyTextures": [0]
+>         }
+>       }
+>     }
+>   ]
+> }
+> ```
+
+
+A `propertyTexture` object extends the glTF [`textureInfo`](../../../../../specification/2.0/schema/textureInfo.schema.json) object. `texCoord` specifies a texture coordinate set in the referring primitive.
+
+The `properties` map specifies the texture channels providing data for available properties. An array of integer index values identifies channels, where multiple channels may be used only for fixed-length arrays of 2, 3, or 4 components. Channels of an `RGBA` texture are numbered 0–3 respectively, although specialized texture formats may allow additional channels. All values are stored in linear space.
+
+> **Example:** A property texture for wind velocity samples. The "speedKPH" property values are stored in the red channel, and "direction" property values are stored as a unit-length vector, with X/Y components in the green and blue channels. Both properties are indexed by UV coordinates in a `TEXCOORD_0` attribute.
+>
+> ```jsonc
+> // Root EXT_mesh_features extension:
+> {
+>   "propertyTextures": [{
+>     "class": "wind",
+>     "index": 0,
+>     "texCoord": 0,
+>     "properties": {
+>       "speedKPH": [0],
```

Comment:
could this just be "speed"?, the exact units aren't important for this example, and it saves some bikeshedding about which way to capitalize `speedKPH` vs `speedKph` etc.

Reply:
Done ✅